.github/workflows/patch-release-build.yml

```diff
@@ -0,0 +1,121 @@
+# Releases a patch by cherrypicking commits into a release branch based on the previous
+# release tag.
+name: Patch Release Build
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The version to tag the release with, e.g., 1.2.1, 1.2.2
+        required: true
+      commits:
+        description: Comma separated list of commit shas to cherrypick
```

Comment:
We're not doing any cherrypicks in this PR yet, is that correct?

Reply:
Nah we're not doing any releasing in this PR, just adding workflows